### PR TITLE
Added support for ATV and ATVMas in albavision plugin

### PR DESCRIPTION
**Pull request by malvinas2**
Replaces PR [https://github.com/streamlink/streamlink/pull/2791](https://github.com/streamlink/streamlink/pull/2791)

### CODE REVIEW

File: src/streamlink/plugins/albavision.py [hunk 1]
```diff
@@ -45,11 +48,14 @@ def page(self):
...
-    def _get_token_url(self):
-        token = self._get_live_url_token()
+    def _get_token_url(self, channelnumber):
+        token = self._get_live_url_token(channelnumber)
         if token:
-            m = self._token_url_re.search(self.page.text)
-            token_url = m and m.group(0)
+            m = self._token_url_re.findall(self.page.text)
+            log.debug("get_token_url m {0}".format(m))
+            log.debug("get_token_url m0 {0}".format(m[channelnumber]))
```
Comment (back-to) [inline src/streamlink/plugins/albavision.py]: there is already `log.debug("token_url={0}{1}".format(token_url, token))`

File: src/streamlink/plugins/albavision.py [hunk 2]
```diff
@@ -45,11 +48,14 @@ def page(self):
...
 
-    def _get_token_url(self):
-        token = self._get_live_url_token()
+    def _get_token_url(self, channelnumber):
+        token = self._get_live_url_token(channelnumber)
         if token:
-            m = self._token_url_re.search(self.page.text)
-            token_url = m and m.group(0)
+            m = self._token_url_re.findall(self.page.text)
+            log.debug("get_token_url m {0}".format(m))
```
Comment (back-to) [inline src/streamlink/plugins/albavision.py]: if you need this `m` maybe use `log.trace`

File: src/streamlink/plugins/albavision.py [hunk 3]
```diff
@@ -66,52 +72,69 @@ def transform_token(token_in, date):
...
         else:
-            live_channel = None
-            for div in itertags(self.page.text, "div"):
-                if div.attributes.get("id") == "botonLive":
-                    live_channel = div.attributes.get("data-canal")
-
             if live_channel:
                 log.debug("Live channel: {0}".format(live_channel))
                 player_url = self._channel_urls[live_channel]+quote(token)
+                log.debug("player url bei live channel {0}".format(player_url))
```
Comment (back-to) [inline src/streamlink/plugins/albavision.py]: not needed **or** use `log.trace` and english language

**Comment by malvinas2** [comment]
Next attempt

**Pull request merged**